
const create = async (req, res) => {
  try {

  } catch (error) {
    res.status(500).json({ error: 'Error en el servidor', details: error })
  }
}
const read = async (req, res) => {
  try {

  } catch (error) {
    res.status(500).json({ error: 'Error en el servidor', details: error })
  }
}
const update = async (req, res) => {
  try {

  } catch (error) {
    res.status(500).json({ error: 'Error en el servidor', details: error })
  }
}
const cancel = async (req, res) => {
  try {

  } catch (error) {
    res.status(500).json({ error: 'Error en el servidor', details: error })
  }
}